amundsen_application/static/js/ducks/bookmark/reducer.ts

```diff
@@ -20,25 +21,25 @@ export function addBookmark(resourceKey: string, resourceType: string): AddBookm
     resourceType,
     type: AddBookmark.REQUEST,
   }
-};
+}
```

Comment:
Did the removal of semicolons come from your editor? If so you may want to look into that setting or if there are extensions for your IDE for the tslint configs we uses. Our tslint configs technically use `semicolon: [true, 'always']`. (however there's been an open task to revisit our lint configs to see why rules that I believed would be applied aren't getting applied).  

Just a nit though, as our linter will eventually enforce consistency.

Reply:
My IDE highlights these semicolons as unnecessary, which means I just do a drive-by cleanup manually. I'll probably have to tinker with my settings to get them to read the tslint config.